### PR TITLE
Completed Verification and Validation of user email address before login.

### PR DESCRIPTION
**Pull request by samimalik325**
Completed Verification and Validation of user email address before login.
User Story: 87 Task: 81, 83

### CODE REVIEW

File: myProject/MovieShowTimeFinder/showtimefinder/views.py [hunk 1]
```diff
@@ -12,24 +20,79 @@
...
-            user = authenticate(username=username, password=raw_password)
-            user.userprofile.dateofbirth = form.cleaned_data['dateofbirth']
-            user.userprofile.zipcode = form.cleaned_data['zipcode']
+            user = form.save(commit=False)
+            user.is_active = False
             user.save()
+            # Email Verification
+            user.userprofile.dateofbirth = form.cleaned_data.get('dateofbirth')
+            user.userprofile.zipcode = form.cleaned_data.get('zipcode')
+            print(user.userprofile.dateofbirth)
```
Comment (ShivamVerma920302) [inline myProject/MovieShowTimeFinder/showtimefinder/views.py]: Remove print statements

File: myProject/MovieShowTimeFinder/showtimefinder/views.py [hunk 2]
```diff
@@ -12,24 +20,79 @@
 import geocoder
 import socket
 import requests
-from urlparams.redirect import param_redirect
+from django.core import serializers
+# from urlparams.redirect import param_redirect
 
 # Create your views here.
```
Comment (ShivamVerma920302) [inline myProject/MovieShowTimeFinder/showtimefinder/views.py]: remove comments in code

File: myProject/MovieShowTimeFinder/showtimefinder/views.py (hunk 1 above)
Comment (samimalik325) [reply myProject/MovieShowTimeFinder/showtimefinder/views.py]: Removed Print Statement

File: myProject/MovieShowTimeFinder/showtimefinder/views.py (hunk 2 above)
Comment (samimalik325) [reply myProject/MovieShowTimeFinder/showtimefinder/views.py]: Removed unnecessary comments

**Pull request merged**